wordpress-mobile/WordPress-Aztec-iOS
# Alpha version of bold editing logic.

Tackles part of #18 

I was planning to split this PR into several PRs, since there are some improvements not-strictly-related to the editing logic.  However since we're two days away from the sprint's end, and there's still [so much to do](https://github.com/wordpress-mobile/WordPress-Aztec-iOS/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Iteration+%233.%22) (and we're not working in production code), I'd rather have this merged and deal with any issues as they arise.

**How to test:**
1. Run the app
2. Go to the editor
3. Select non-bold text and press the B button in the toolbar.
4. Switch to HTML mode and make sure the tag is added to the selected text.

**What doesn't work:**
1. Removing bold
2. The way in which bold is added is kinda primitive now - there's no logic to merge adjacent bold elements into a single one, so the results are a lot of bold tags everywhere.  This is known and accepted for this alpha.
